velox/functions/sparksql/tests/ComparisonsTest.cpp

```diff
@@ -86,9 +122,63 @@ TEST_F(ComparisonsTest, equaltonullsafe) {
   EXPECT_EQ(equaltonullsafe<double>(kNaN, std::nullopt), false);
   EXPECT_EQ(equaltonullsafe<double>(kNaN, 1), false);
   EXPECT_EQ(equaltonullsafe<double>(kNaN, kNaN), true);
+
+  runAndCompareForArrayInput(funcName, "null", "null", true);
```

Comment:
It seems funcName repeats many times. Can we add a lambda function like `assertArrayInput("null", "null", true)` to replace?